Makefile

```diff
@@ -2,7 +2,7 @@ PROFILE?=
 # Defines the product that the test aims to test
 # Since we already have test for RHCOS4, this is the default for now.
 PRODUCT?=rhcos4
-CONTENT_IMAGE?=quay.io/complianceascode/ocp4:latest
+CONTENT_IMAGE?=quay.io/compliance-operator/compliance-operator-content:latest
```

Comment:
Mind adding docs on how this is built and where it comes from?

Reply:
Yeah - good call. I already have a patch up that adds a README.md 

https://github.com/ComplianceAsCode/ocp4e2e/pull/25

Can I stack it on top of that? I already proposed a patch to openshift/compliance-operator that describes the container images.

https://github.com/openshift/compliance-operator/pull/824

Reply:
Thanks for adding those! As a last piece...  is there automation in place to build the `ocp4` container tag? or is that being done manually?

Reply:
Can you elaborate on the `ocp4` container tag bit? We're tagging container images with the git short SHA, branch, and `latest` (when applicable).